tensorboard_plugin_wit/wit_plugin.py

```diff
@@ -318,7 +336,8 @@ def _infer(self, request):
             model_signatures[model_num],
             request.args.get('use_predict') == 'true',
             request.args.get('predict_input_tensor'),
-            request.args.get('predict_output_tensor'))
+            request.args.get('predict_output_tensor'),
+            custom_predict_fn=self.custom_predict_fn)
```

Comment:
also add this in the serving bunding constructor call in the infer_mutants method, otherwise partial dependence plots don't work with this custom predict fn

Reply:
I tested my modified code and found that the graph loads without a problem only if I choose "regression" in the setup page, is that expected?

Reply:
if your custom prediction fn returns a list of lists, with the inner list having a score per class, then the PD plots should work fine when regression is un-checked (as this model is returning classification scores). If your custom fn just returns a  list of scores, then it is for a regression model and the tool (including PD plots) should work correctly when the UI is set in "regression" mode through the checkbox.

Are you seeing this behavior or something different?

Reply:
Hi, If I return a list of list and choose "classification", there's a toast message says "Request for plot creation failed: TypeError: Cannot read property 'map' of undefined". The same custom predict function (returns a list of list) works correctly in the default (Datapoints) plot.

Reply:
and "regression" with a custom predict function returning a list of scores works fine.

Reply:
Strange, with the following custom predict fn and default WIT settings (so, binary classification), I was able to see correct PD plots.

```
import random

def custom_predict_fn(examples, serving_bundle):
  number_of_examples = len(examples)
  
  classifications_for_all_examples = []
  for _ in range(number_of_examples):
    score = random.random()
    classifications_for_all_examples.append([1 - score, score])
  return classifications_for_all_examples
```

Reply:
Is this resolved?

Reply:
i see the PR as working as expected. we will also put all features, including this, through manual testing before our next pip release